certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/common/codesystems/CodeSystemKvTs001.java

```diff
@@ -24,6 +24,12 @@ public class CodeSystemKvTs001 {
       "Vet ej"
   );
 
+  public static final Code NO_KNOWLEDGE_V2 = new Code(
```

Comment:
Suggestion is that we remove the V2 in the name. Reason is that from the code system perspective it is just another code and different models, that want to use the code system, can decide what codes to use.

Reply:
Updated to Swedish names to differ ej / inte